mlflow/tracking/fluent.py

```diff
@@ -386,6 +388,25 @@ def active_run() -> Optional[ActiveRun]:
     return _active_run_stack[-1] if len(_active_run_stack) > 0 else None
 
 
+def last_active_run() -> Optional[Run]:
+    """
+    Returns mlflow.active_run() if it is not None. Otherwise, return the
+    last run started from the current Python process that reached a
+    terminal status (i.e. FINISHED, FAILED, or KILLED).
+
+    This is useful for retrieving the most recent autologged run.
```

Comment:
Let's clarify that this is just one example where this is useful. Can we also include some example code?

Reply:
Can we push changes before marking this resolved?